packages/ui/QuestionFormInput/index.tsx

```diff
@@ -459,7 +478,7 @@ export const QuestionFormInput = ({
           />
         )}
       </div>
-      {selectedLanguageCode !== "default" && value && value["default"] && (
+      {selectedLanguageCode !== "default" && value && (value["default"] || value["default"] === "") && (
```

Comment:
Can this condition also be simplified if possible for readability? 
Maybe GPT says: `selectedLanguageCode !== "default" && value?.["default"] ?? "" !== "";` but can you test & see if it works and no corner case is missed?